specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2024-03-01/AutomationRules.json

```diff
@@ -987,11 +1295,28 @@
         }
       }
     },
-    "PlaybookActionProperties": {
+    "ManualTriggerRequestBody": {
+      "required": [
+        "logicAppsResourceId"
+      ],
       "type": "object",
+      "properties": {
+        "tenantId": {
+          "format": "uuid",
+          "description": "The tenant id of the playbook resource.",
+          "type": "string"
+        },
+        "logicAppsResourceId": {
+          "description": "The resource id of the playbook resource.",
+          "type": "string"
+        }
+      }
+    },
+    "PlaybookActionProperties": {
       "required": [
         "logicAppResourceId"
       ],
+      "type": "object",
       "properties": {
         "logicAppResourceId": {
           "description": "The resource id of the playbook resource.",
```

Comment:
>description [](http://example.com/codeflow?start=11&length=11)

"format": "arm-id"?
And perhaps [https://azure.github.io/autorest/extensions/#x-ms-arm-id-details](https://azure.github.io/autorest/extensions/#x-ms-arm-id-details)